src/python/pants/core/goals/fmt.py

```diff
@@ -145,11 +155,22 @@ def register_options(cls, register) -> None:
                 "faster than `--no-per-file-caching` for your use case."
             ),
         )
+        register(
+            "--batch-size",
+            advanced=True,
+            type=int,
+            default=128,
```

Comment:
Does this __need__ to be a power of two?

Reply:
No.